ledger/participant-state/kvutils/src/test/suite/scala/com/daml/ledger/participant/state/kvutils/api/KeyValueParticipantStateWriterSpec.scala

```diff
@@ -43,9 +43,10 @@ class KeyValueParticipantStateWriterSpec extends WordSpec with Matchers {
       instance.submitTransaction(
         submitterInfo(recordTime, aParty, expectedCorrelationId),
         transactionMeta(recordTime),
-        anEmptyTransaction)
+        anEmptyTransaction,
+        Long.MaxValue)
```

Comment:
Since this seems unused, why not passing a shorter value (or `None`, if it can become optional)?

Reply:
That value doesn't matter from the test's perspective -- changed.